resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -182,6 +182,16 @@ private[spark] object Config extends Logging {
       .checkValue(_ >= 0, "The minimum number of tasks should be non-negative.")
       .createWithDefault(0)
 
+  val EXECUTOR_ROLL_OUTLIERS_ONLY =
+    ConfigBuilder("spark.kubernetes.executor.onlyRollOutliers")
+      .doc("Only roll an executor if it is an outlier. An outlier is defined to be at least two " +
+        "standard deviation outside of the mean. If no outlier is found then no executor will " +
+        "be rolled."
+      )
```

Comment:
Please place `)` at line 189.

Reply:
I removed the config so I'm going to resolve this.